apps/backend/user/src/verifyMFA.ts

```diff
@@ -0,0 +1,18 @@
+import { User } from "./models/user";
+import { TOTP } from "otpauth";
+
+const verifyMFA = async (user: User, totp_secret: string) => {
+  // Create TOTP from stored secret
+  const totp = new TOTP({
+    label: "Martletplace",
+    algorithm: "SHA1",
+    digits: 6,
+    secret: user.totp_secret!,
+  });
+
+  const isValid = totp.validate({ token: totp_secret });
```

Comment:
Should this var be named `isValid` then if its not a bool or 0/1? What's the possible values for it?

Reply:
That validate method returns the token delta or possibly null if very invalid, so I've renamed the const variable to better reflect that.